README.md

```diff
@@ -38,7 +38,7 @@ The EditorConfig Vim plugin supports the following EditorConfig [properties][]:
 * `tab_width`
 * `end_of_line`
 * `charset`
-* `insert_final_newline` ([PreserveNoEOL][] is required for this property)
+* `insert_final_newline` (Vim 7.4.785 or later or [PreserveNoEOL][] is required for this property)
```

Comment:
I would recommend using the statement like the following:

    Feature +fixendofline is enabled which is available on Vim 7.4.785+, or [PreserveNoEOL]() ...

What do you think?